Demo/Application/Base/AppDelegate.swift

```diff
@@ -4,7 +4,8 @@ import BraintreeCore
 @UIApplicationMain class AppDelegate: UIResponder, UIApplicationDelegate {
         
     private let returnURLScheme = "com.braintreepayments.Demo.payments"
-    private let universalLinkURL = "https://braintree-ios-demo.fly.dev/braintree-payments"
```

Comment:
q: Is this the domain that hosts the `apple-app-site-association` file?

Reply:
Yes! Our Demo app now has an HTTPS URL registered for it 🎊 - https://braintree-ios-demo.fly.dev/.well-known/apple-app-site-association

Reply:
Jax built it out https://github.com/jaxdesmarais/fly-server-demo

Reply:
Nice that's cool!